scripts/update-default-opts.sh

```diff
@@ -8,16 +8,9 @@
 begin="BEGIN_DEFAULT_OPTS"
 end="END_DEFAULT_OPTS"
 
-# scrape DEFAULT_OPTS, indented at 2
```

Comment:
This is needed to generate `/tmp/DEFAULT_OPTS.6.lua`

Reply:
Fixed https://github.com/kyazdani42/nvim-tree.lua/pull/1327/commits/e46b7571de4935fa06aca3e6bc487fe17837f5c1

Reply:
Thanks @alex-courtis, should have tested it locally. Once this PR is merged in I can work on GitHub action generating help for each PR adding  new option - should be very straight forward.